tests/layer_tests/common/legacy/utils/common_utils.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import os
+import subprocess
+import sys
+
```

Comment:
I don't think that 'legacy' folder is necessary

Reply:
Let`s use only common/utils folder

Reply:
I meant deleting only the 'legacy' folder, the 'utils' folder should`t have been deleted